### PR TITLE
Add github actions to dependabot, auto merge on green for dependabot PRs

### PR DESCRIPTION
**Pull request by jpkrohling**
Signed-off-by: Juraci Paixão Kröhling <juraci@kroehling.de>

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -4,3 +4,8 @@ updates:
     directory: "/"
     schedule:
       interval: daily
+
+  - package-ecosystem: "github-actions"
```
Comment (objectiser) [inline .github/dependabot.yml:8]: Do the quotes here and on the `internal` below matter, as they are not used in the above configuration?

Comment (jpkrohling) [reply .github/dependabot.yml:8]: No, copy/paste from the otel-operator, which likely added each one of them at different times :-)

**Review by objectiser — APPROVED** [review]

**Comment by jpkrohling** [comment]
Restarting the smoke-otel job, failure in the sidecar test.

**Comment by jpkrohling** [comment]
The same test failed again. @kevinearls, is this a known issue?

**Comment by kevinearls** [comment]
@jpkrohling not that I am aware of, although I think that test has been a bit flaky in the past.  @jkandasa have you seen this?  If not we need to investigate.

**Comment by jkandasa** [comment]
@kevinearls These days I do not run `otel` tests. But sometimes I see failure on different tests. rerun passes those.

**Comment by jpkrohling** [comment]
Looks like #1377 is also failing because of that. So, either master is broken, or the test is really unreliable these days.

**Comment by jpkrohling** [comment]
I just ran it locally, and it does indeed fail. Here's what I can see in one of the logs:
```
 $ kubectl logs -n osdk-e2e-6a0aaaed-86e6-417d-b86d-8cda6db5a082 osdk-e2e-6a0aaaed-86e6-417d-b86d-8cda6db5a082-7f777f5c77-btq2t -c jaeger-agent
Error: unknown flag: --jaeger.tags
2021/02/03 11:07:56 Failed to run the service: unknown flag: --jaeger.tags
```

This seems to have been recently merged in the main repository. Are we using `latest` images as part of the e2e tests?

**Comment by kevinearls** [comment]
@jpkrohling Yes, the tests default to using latest

**Comment by jpkrohling** [comment]
The current merge queue is then blocked by #1378.

**Pull request merged**